python/lsst/analysis/tools/actions/vector/mathActions.py

```diff
@@ -24,8 +24,13 @@
     "ConstantValue",
     "AddVector",
     "SubtractVector",
-    "DivideVector",
     "MultiplyVector",
+    "DivideVector",
+    "SquareVector",
+    "SqrtVector",
+    "ExponentiateFromBaseVector",
+    "ExponentiateToPowerVector",
```

Comment:
"Exponentiate"? I realize this is the correct usage of the word here, but it feel overly verbose. Can we agree to use `ExpFromBaseVector` and `ExpToPowerVector` and pretend that "Exp" stands for "Exponentiate"?

Reply:
"exp" implies e^x, which is not what this is. The other alternative I considered was `RaiseToPower`. I didn't like that partly because raise is a Python keyword, and I'm not sure if `RaiseFromBase` makes sense.

Reply:
Exponentiate is a confusing term, I think (at least, it is to me). I *slightly* prefer `RaiseToPower` and `RaiseFromBase`, but I accept your point that `Raise` is a protected word in other contexts. Although perhaps that's not an issue here?